ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -92,41 +94,140 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(__unused RCTPromiseRejectBlock)reject)
 {
-  UILocalNotification *notification = [self _localNotificationFromPayload:payload];
+  UNMutableNotificationContent* content = [self _localNotificationFromPayload:payload];
 
   [EXUtil performSynchronouslyOnMainThread:^{
-    [RCTSharedApplication() presentLocalNotificationNow:notification];
+    UNNotificationRequest* request = [UNNotificationRequest
+                                      requestWithIdentifier:content.userInfo[@"id"] content:content trigger:nil];
```

Comment:
Either make this method call in one line or go with `[UNNotificationRequest requestWithIdentifier:…NEWLINE`